### PR TITLE
Changelog for 0.9.5 release

### PR DESCRIPTION
**Pull request by esantorella**
## Motivation

Changelog for 0.9.5 release

## Test Plan

* Looked at the file in a markdown viewer
* Running the nightly cron to make sure everything is green before we put out a release: https://github.com/pytorch/botorch/actions/runs/7145907153

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,34 @@
 
 The release log for BoTorch.
 
+## [0.9.5] -- Dec 8, 2023
+
+#### New features
+
+Hypervolume Knowledge Gradient (HVKG):
+* Add `qHypervolumeKnowledgeGradient`, which seeks to maximize the difference in hypervolume of the hypervolume-maximizing set of a fixed size after conditioning the unknown observation(s) that would be received if X were evaluated (#1950).
+* Add initializer vor one-shot HVKG (#1982).
```
Comment (saitcakmak) [inline CHANGELOG.md:11]:
```suggestion
* Add initializer for one-shot HVKG (#1982).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,34 @@
 
 The release log for BoTorch.
 
+## [0.9.5] -- Dec 8, 2023
+
+#### New features
+
+Hypervolume Knowledge Gradient (HVKG):
+* Add `qHypervolumeKnowledgeGradient`, which seeks to maximize the difference in hypervolume of the hypervolume-maximizing set of a fixed size after conditioning the unknown observation(s) that would be received if X were evaluated (#1950).
```
Comment (saitcakmak) [inline CHANGELOG.md:10]: I find this sentence a bit hard to follow but looks like this is also what is in the docstring. Probably ok to leave as is.

**Review by saitcakmak — APPROVED** [review]
Thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,34 @@
...
+* Add initializer for one-shot HVKG (#1982).
+* Add tutorial on decoupled Multi-Objective Bayesian Optimization (MOBO) with HVKG (#2094).
+* Illustrate how to use Multi-Fidelity HVKG (MV-HVKG) (#2101).
```
Comment (Balandat) [inline CHANGELOG.md:13]: It feels like rather than having a line item for each PR, can we logically group the PRs together? 
```suggestion
* Add `qHypervolumeKnowledgeGradient`, which seeks to maximize the difference in hypervolume of the hypervolume-maximizing set of a fixed size after conditioning the unknown observation(s) that would be received if X were evaluated (#1950, #1982, #2101).
* Add tutorial on decoupled Multi-Objective Bayesian Optimization (MOBO) with HVKG (#2094).
```
